source/ns_cmdline.c

```diff
@@ -215,7 +215,7 @@
 #endif
 // Maximum length of input line
 #ifndef MBED_CONF_CMDLINE_MAX_LINE_LENGTH
-#define MBED_CONF_CMDLINE_MAX_LINE_LENGTH 2000
+#define MBED_CONF_CMDLINE_MAX_LINE_LENGTH 200
```

Comment:
why this default change?

Reply:
To optimise RAM consumption: https://github.com/ARMmbed/mbed-os/issues/13438

Reply:
Ok, well its probably a bit of a breaking change. At least for our testing :-)

Reply:
well, not a big deal. Easy to configure it back to larger value for testing.